venues/ICLR.cc/2018/Conference/webfield/programchairWebfield.js

```diff
@@ -0,0 +1,815 @@
+
+// Assumes the following pattern for meta reviews and official reviews:
+// CONFERENCE + '/-/Paper' + number + '/Meta_Review'
+// CONFERENCE + '/-/Paper' + number + '/Official_Review'
+
+// Constants
+var HEADER_TEXT = 'ICLR 2018 Program Chairs Console';
```

Comment:
Constants are the ICLR related things. It would be nice to save this in a template, what do you think?.